tests/Controller/Api/PostControllerTest.php

```diff
@@ -360,9 +362,17 @@ protected function createPostController($postManager = null, $commentManager = n
             $formFactory = $this->createMock('Symfony\Component\Form\FormFactoryInterface');
         }
         if (null === $formatterPool) {
-            $formatterPool = $this->createMock('Sonata\FormatterBundle\Formatter\Pool');
+            $formatterPool = $this->getPoolMock();
         }
 
         return new PostController($postManager, $commentManager, $mailer, $formFactory, $formatterPool);
     }
+
+    /**
+     * @return mixed Pool|PoolInterface
+     */
+    private function getPoolMock()
```

Comment:
Add a next major tag to remove this, we will probably drop < formatter 4.x on next major

Reply:
Done